firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -34,14 +34,16 @@ public expect val Firebase.firestore: FirebaseFirestore
 /** Returns the [FirebaseFirestore] instance of a given [FirebaseApp]. */
 public expect fun Firebase.firestore(app: FirebaseApp): FirebaseFirestore
 
-public expect class NativeFirebaseFirestore
+internal expect class NativeFirebaseFirestore
```

Comment:
This is a typealias to `FIRFirestore`(ios) and `com.google.firebase.firestore.FirebaseFirestore`(android) or the wrapper on Javascript. To remove it from the public API, it is now internal but this has a cascading effect on
https://github.com/GitLiveApp/firebase-kotlin-sdk/pull/555/files#r1655371860
https://github.com/GitLiveApp/firebase-kotlin-sdk/pull/555/files#r1655371283